pkgs/development/libraries/unixODBCDrivers/default.nix

```
@@ -29,35 +29,42 @@
 
   mariadb = stdenv.mkDerivation rec {
     pname = "mariadb-connector-odbc";
-    version = "3.1.2";
-
-    src = fetchurl {
-      url = "https://downloads.mariadb.org/interstitial/connector-odbc-${version}/${pname}-${version}-ga-src.tar.gz";
-      sha256 = "0iibly2mbqijqyq4pzpb6dh40clqhvqrhgnj8knm4bw3nlksd0d5";
+    version = "3.1.4";
+
+    src = fetchFromGitHub {
+      owner = "MariaDB";
+      repo = "mariadb-connector-odbc";
+      rev = "c921b500650eca535010f12e0d83417d2ef915ec";
```

I'm terribly sorry I didn't notice this before... we generally set `rev` to something involving `version` for easy updates. So in this case:

```suggestion
      rev = version;
```

I've tested this on NixOS and after this change I'm happy to merge to `master`. This change deserves a backport as well. Thank you so much for digging into this! Hopefully at some point someone can get to the bottom of our `libmysqlclient` issue, but until that happens the priority is to have a working package :smile:

Thanks again @jamii :tada: